### DISCUSSION TITLE
Unversioned files?

### DISCUSSION THREAD

**Discussion by TreyWW** [jimporter/mike#214] [Q&A]
Hi,

Is there a way to set certain files to be unversioned. Things like a `contributors list`, or `project motivation`, etc etc, some of these pages I don't want in a specific version, I want these always to be the latest.

Is there a way to allow these to still be e.g. `/motivation`, instead of `/v0.0.2/motivation`, or at least a way to get the most up to date `/motivation` no matter what version you're on?

Would be a nice feature if this isn't yet added

**Comment by TreyWW** [comment]
Aha nevermind, this was covered on a closed issue, didn't originally see it. Apologies.

https://github.com/jimporter/mike/issues/158

**Comment by TreyWW** [comment]
Hmm, I haven't been able to figure out how to use this. I attempted to create a folder `versioned` inside of docs, and then added deploy_prefix to my plugins section

```
  - mike:
      deploy_prefix: 'versioned'
```

But I just get `Message: File not found.`, so it's no longer using my unversioned files like `index.md` etc, what config am I missing to allow both non-versioned and versioned files on the same docs? Thanks

**Reply by jimporter** [reply]
The `deploy_prefix` is just the subdirectory in your `gh-pages` branch to copy your generated docs to. Where you put your source `.md` files is totally separate, and doesn't have anything to do with `deploy_prefix`.

**Reply by TreyWW** [reply]
Yeah that's what I guessed, how can I achieve this then, is it not possible?

**Reply by jimporter** [reply]
It's possible, but I don't know what your configuration looks like so I can't be very specific. This discussion goes over some of the ideas though: https://github.com/jimporter/mike/discussions/206

**Reply by TreyWW** [reply]
Cool thanks, seems like a complicated answer but i'll give it a shot. Appreciate it :)
